client/src/telemetry/glean-context.tsx

```diff
@@ -101,8 +101,8 @@ function glean(): GleanAnalytics {
 
   if (DEV_MODE) {
     Glean.setDebugViewTag("mdn-dev");
+    Glean.setLogPings(GLEAN_DEBUG);
   }
-  Glean.setLogPings(GLEAN_DEBUG);
```

Comment:
I'm not sure what the intent was here, but we currently support turning on/off debugging from the browser console. By having this line here, whatever we set from the console gets overridden by `GLEAN_DEBUG`

Reply:
@Dexterp37 I'm curious, how can I enable debugging from the console?

Reply:
@caugner good question! By opening the browser console, you can do `window.Glean.set*` (you have 3 functions there that you can use to live debug production things as well!)